ChangeLog

```diff
@@ -1,5 +1,12 @@
  # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
+3.41 2020-10-09 Karl Williamson <khw@cpan.org>
```

Comment:
Why is this commit here? Hasn't 3.41 already been released?

Reply:
Apparently, the administrative changes didn't get pushed to master at the proper time.

Reply:
Things have been fixed up as a result of this comment